cli/helm/chart_test.go

```diff
@@ -31,6 +32,20 @@ func TestLoadChart(t *testing.T) {
 	require.Equal(t, expectedValues, actual.Values)
 }
 
+func TestFetchChartValues(t *testing.T) {
+	namespace := "default"
+	name := "consul"
+	settings := mock.CreateMockEnvSettings(t, namespace)
+	logger := mock.CreateLogger(t)
+
+	expected := map[string]interface{}{}
+
+	actual, err := FetchChartValues(namespace, name, settings, logger)
+	require.NoError(t, err)
+
+	require.Equal(t, expected, actual)
+}
+
```

Comment:
@ishustava, this test provides coverage for the fundamental bug being fixed. We definitely need a better foundation for testing behavior of installs, etc. but once this test is finished, I think it will be enough to cover what is necessary.